### DISCUSSION TITLE
SLES 15 SP1 anyone ?

### DISCUSSION THREAD

**Discussion by fperillo** [vmware/packer-examples-for-vsphere#794] [Help] [answered]
We have a software that before installing check for os version and doesn't install on anything newer than SLES15 SP1.

So I was tasked to create a template... 

I installed a basic SP1, dumped the autoinst.xml 

Then started to use this tool and, after adding netsetup=dhcp otherwise it din' setup the network card, I get errors from the sles install process, reporting a problem with the autoinst.xml file.

I will investigate more tomorrow, and will also try a couple of autoinst.xml samples found on the web. 

But I'd ask here 2 questions:

- is autoinst.xml sent as is to installing server or is modified somehow? 
- has anybody succesfully built a SLES15 SP1 template ?

Thank you

**Comment by tenthirtyam** [comment, marked as answer]
If you need to use 15.1 you'll of course need to use that ISO but also update the `autoinst.pkrtl.hcl` template from to set 15.1.

See lines:

https://github.com/vmware-samples/packer-examples-for-vsphere/blob/1ee78059240798c384d5e6308a49843b77d29c50/builds/linux/sles/15/data/autoinst.pkrtpl.hcl#L767

https://github.com/vmware-samples/packer-examples-for-vsphere/blob/1ee78059240798c384d5e6308a49843b77d29c50/builds/linux/sles/15/data/autoinst.pkrtpl.hcl#L773
